frontend/src/lib/lemon-ui/Spinner/Spinner.scss

```diff
@@ -1,4 +1,5 @@
 .Spinner {
+    --speed: 1s;
```

Comment:
Let's use `--spinner-speed` for clarity, matching `--spiner-color`

Reply:
100%

Reply:
Done, see 0bcdf527e217561c2cddac7ec759b8268ce61b9f